core/jobs/types/firebase_domain_test.py

```
@@ -0,0 +1,162 @@
+
+    def test_from_export_with_disabled_record(self) -> None:
+        export_record = mock.Mock(uid='uid', email='a@a.com', disabled=True)
+        record = firebase_domain.FirebaseRecord.from_export(export_record)
+        self.assertEqual(record.auth_id, 'uid')
+        self.assertEqual(record.email, 'a@a.com')
+        self.assertTrue(record.disabled)
+
+    def test_from_export_with_missing_fields_raises_value_error(self) -> None:
+        with (
+            self.subTest('email is None'),
+            self.assertRaisesRegex(ValueError, r'needs non-empty'),
```

Ruff (0.15.17)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)